packages/react-native-branch/android/src/main/java/expo/modules/adapters/branch/BranchPackage.kt

```diff
@@ -0,0 +1,15 @@
+package expo.modules.adapters.branch
+
+import android.content.Context
+import expo.modules.core.BasePackage
+import expo.modules.core.interfaces.ApplicationLifecycleListener
+import expo.modules.core.interfaces.ReactActivityLifecycleListener
+
+class BranchPackage : BasePackage() {
```

Comment:
```suggestion
class BranchPackage : Package {
```

we are planning to deprecate `BasePackage` and use the Package directly with java 8 interface default method.